api-objects/patientCreateResponse.ts

```diff
@@ -0,0 +1,5 @@
+export interface PatientCreateResponse {
+    patientid: number;
+    deceased: boolean;
+    patientnotes: string;
+}
```

Comment:
Not that this would make a difference all that much, but I feel that a lot of this patient information is going to be duplicated as we add more API calls for patient data. Would it be better to have just a generic patient data object? For now it's pedantic, but I worry it might set a precedent down the road.

Reply:
Good point, I'll rename this file; however, I think there will be some situations where we need to specify specific data that either the API or the client will expect.

Reply:
That's a fair point